depstubber.go

```diff
@@ -150,6 +171,55 @@ func createStubs(packageName string, typeNames []string, funcAndVarNames []strin
 	if _, err := dst.Write(g.Output()); err != nil {
 		log.Fatalf("Failed writing to destination: %v", err)
 	}
+
+	if licenseDirs != nil {
+		for _, licenseSearchDir := range licenseDirs {
+			fl, err := filer.FromDirectory(licenseSearchDir)
+			if err != nil {
+				panic(err)
+			}
+			licenses, err := licensedb.Detect(fl)
+			if err != nil {
+				panic(err)
+			}
+
+			for _, licenseRelativePath := range gatherFilenames(licenses) {
```

Comment:
Why isn't this just
```suggestion
			for licenseRelativePath := range licenses {
```

Reply:
Because that's not the filenames; it's the license type. We need the filenames.

`licenses` =

```
(map[string]api.Match) (len=4) {
 (string) (len=7) "ECL-2.0": (api.Match) {
  Files: (map[string]float32) (len=1) {
   (string) (len=7) "LICENSE": (float32) 0.8988327
  },
  Confidence: (float32) 0.8988327
 },
 (string) (len=7) "SHL-0.5": (api.Match) {
  Files: (map[string]float32) (len=1) {
   (string) (len=7) "LICENSE": (float32) 0.85278857
  },
  Confidence: (float32) 0.85278857
 },
 (string) (len=8) "SHL-0.51": (api.Match) {
  Files: (map[string]float32) (len=1) {
   (string) (len=7) "LICENSE": (float32) 0.85343707
  },
  Confidence: (float32) 0.85343707
 },
 (string) (len=10) "Apache-2.0": (api.Match) {
  Files: (map[string]float32) (len=1) {
   (string) (len=7) "LICENSE": (float32) 0.9851036
  },
  Confidence: (float32) 0.9851036
 }
}
```

Reply:
Er, yes, That was a rather poor comment.

What I'd meant is could you just move the `v.Files` access into this loop so you're not reallocating everything?

Reply:
Done.